data/sensors_recent.php

```diff
@@ -278,11 +381,16 @@ function output_cell($rowspan, $data) {
 					$url = "http://www.openstreetmap.org/?mlat=" . $row['latitude'] . "&amp;mlon=" . $row['longitude'];
 					output_cell($rowspan, "<a href=\"" . $url . "\">" . $row["latitude"] . " / " . $row["longitude"] . "</a>");
 				}
+
 				if (array_key_exists('counter', $message)) {
 					output_cell($rowspan, $message['counter']);
+				} else if (array_key_exists('f_cnt', $message['uplink_message'])) {
+					output_cell($rowspan, $message['uplink_message']['f_cnt']);
```

Comment:
Hier ben je eigenlijk afhankelijk van ttn.v1/v2 vs ttn.v3 een andere element uit `$message` aan het pakken voor de frame counter. Het zou denk ik netter zijn om alle ttn.v1/v2/v3 code naar bovenaan te verplaatsen en daar `$metadata['counter']` oid te vullen, zodat je hier gewoon direct alleen die waarde hoeft te gebruiken. Volgens mij betekent dat ook dat je alleen in de v1/v2/v3 code bovenaan iets met `$message` hoeft te doen, en je daarna alleen nog maar naar `$row`, `$metadata` en `$gateways` hoeft te kijken.